e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('I18456 Built-in slash commands: common', () => {
+    before(() => {
+        loginAndVisitDefaultChannel('user-1');
+    });
+
+    it('/ autocomplete list can scroll', () => {
+        // # Clear post textbox
+        cy.get('#post_textbox').clear().type('/');
+
+        // * Suggestion list should be visible
+        // # Scroll to bottom and verify that the last command "/shrug" is visible
+        cy.get('#suggestionList').should('be.visible').scrollTo('bottom').then((container) => {
+            cy.getByText(/\/away/, {container}).should('not.be.visible');
+            cy.getByText(/\/shrug/, {container}).should('be.visible');
+        });
+
+        // # Scroll to top and verify that the first command "/away" is visible
+        cy.get('#suggestionList').scrollTo('top').then((container) => {
+            cy.getByText(/\/away/, {container}).should('be.visible');
+            cy.getByText(/\/shrug/, {container}).should('not.be.visible');
+        });
+    });
+
+    it('/shrug test', () => {
+        // # Login as user-2 and post a message
+        loginAndVisitDefaultChannel('user-2');
+        cy.postMessage('hello from user-2');
```

Comment:
In order to save some time, instead of posting a message from UI, can we make an API request